pkg/controller/azure/storage/account/account.go

```diff
@@ -74,6 +74,8 @@ var (
 	resultRequeue    = reconcile.Result{Requeue: true}
 	requeueOnSuccess = reconcile.Result{RequeueAfter: requeueAfterOnSuccess}
 	requeueOnWait    = reconcile.Result{RequeueAfter: requeueAfterOnWait}
+
+	log = logging.Logger.WithName("controller." + controllerName)
 )
 
 // Reconciler reconciles a GCP storage account acct
```

Comment:
Not part of this PR, :) - GCP needs to be removed (I missed it in the original PR)